docs/source/user_guide/groups.rst

```diff
@@ -1101,15 +1093,18 @@ shown below under the `plugins.3d` key of your
                 // Whether to show the 3D visualizer
                 "enabled": true,
 
-                // The initial camera position in 3D space
+                // The initial camera position in the 3D scene
                 "defaultCameraPosition": {"x": 0, "y": 0, "z": 0},
 
                 // Transformation from PCD -> reference coordinates
                 "pointCloud": {
                     // A rotation to apply to the PCD's coordinate system
                     "rotation": [0, 0, 0],
 
-                    // Don't render points below this z value
+                    // Define the lower z extent of the "color by height" range
```

Comment:
To me, this first line makes it sound like points below `minZ` will still be rendered but will just have the same color as `minZ`. But later you clarify that this does indeed cull points below `minZ`.

Reply:
Ah that is confusing. Removing that line.

Reply:
See https://github.com/voxel51/fiftyone/pull/2379/files